### DISCUSSION TITLE
wmbus signal strength

### DISCUSSION THREAD

**Discussion by cukoun** [wmbusmeters/wmbusmeters#823] [General]
Hi,
i have been experiencing signal issues. I installed my iM871A USB dongle to the RPI4 with HA. My RPI4 is closed to the metal rack where is also all my network and wifi stuff. When I close metal door of my rack all of my meters (water and heat) probably still sending datagrams but on the wmbusmeters logs I did not seeing any traffic :-(. Everything is working fine when I open this metal door :-) :-).
Distance between dongle and all meters is around 5-6 meters. Did someone experienced something similar?

Thx
Tom

**Comment by lindsand** [comment]
Try to use an USB extension cord (Male A to Female A) and place the USB dongle outside the metal rack!

**Comment by kzajac83** [comment]
https://en.wikipedia.org/wiki/Faraday_cage
